tests/gnrc_dhcpv6_client_6lbr/README.md

```diff
@@ -24,7 +24,7 @@ The tests succeeds if you see the string `SUCCESS`.
 
 If any problems are encountered (i.e. if the test prints the string `FAILED`),
 set the echo parameter in the `run()` function at the bottom of the test script
-(tests/01-run.py) to `True`. The test script will then offer a more detailed
+(tests-as-root/01-run.py) to `True`. The test script will then offer a more detailed
```

Comment:
@miri64 Thanks :) While I was at it I corrected the path to the test files in this line and also squashed it already. I hope that this was okay